### ISSUE TITLE
Design for empty Tests due state

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-anc#474]
![Screenshot_20200225-100640_WHO ANC](https://user-images.githubusercontent.com/3488986/75223179-f425be00-57b6-11ea-98b9-c815221809bc.jpg)

If no tests are due, the page is blank.

**Comment by rowo** [comment]
@cafootitt if I remember correctly, the reason this exists is because there exists a technical limitation where a user has to go here even if there are no tests and no other questions to answer — is that correct?

**Comment by cafootitt** [comment]
@rowo Yes, I believe you're correct. There's also the Other Tests section that will always have tests listed there, but they are not due. This tab is only showing the due tests.
